pkg/apiserver/handlers/featuregates/handler.go

```diff
@@ -74,8 +68,16 @@ func HandleFunc(k8sclient clientset.Interface) http.HandlerFunc {
 			return
 		}
 
-		agentfeatureGates := getAgentGatesResponse(agentConfig)
-		controllerfeatureGates := getControllerGatesResponse()
+		controllerConfig := &Config{}
+		err = yaml.Unmarshal([]byte(antreaConfig.Data["antrea-controller.conf"]), controllerConfig)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			klog.Errorf("Failed to unmarshal Antrea antrea-controller.conf: %v", err)
```

Comment:
For new added code, use structured log like `klog.ErrorS`.

Reply:
fix: https://github.com/antrea-io/antrea/pull/5314/commits/22868c37f382a0a3b45d049681f5f55021ccc59b

Reply:
Just curious, Why do we use `klog.ErrorS` for newly added code?  And will we change it to `klog.Errorf` in the future?

Reply:
`klog.ErrorS` is the new approach, as we (just like the K8s codebase) are moving to structured logs. Structured logs have a few advantages: easier to consume in monitoring tools, more uniformity when the same object is referenced in multiple logs, etc.
We ask every contributor to use structured logging primitives in new code, and upgrade to structured logging primitives when they modify an existing function or file.

Reply:
@antoninbas Thanks for your explanation.